### ISSUE TITLE
Support translation for documents

### ISSUE THREAD

**Issue by janno42** [fsr-de/1327#665]
Documents should be translatable (German and English)

**Comment by karyon** [comment]
we encountered the following problem: @hendraet wants to add a field to the userprofile:
```
# user_management/migrations/0003_userprofile_language.py
migrations.AddField(
            model_name='userprofile',
            name='language',
...
```
however, there are multiple function from django guardian (get_anonymous_user, create_anonymous_user), which access the model directly, and not through `apps.get_model`, as you need to do in migrations. as a result, when working on the database these methods assume that the new `language` column exists, but it does not (yet), since the respective migration has not run yet.

we can not work around this with setting some dependencies, since the preceding migration to the one adding the field does also use one of these methods.

options:
1. bug report to django-guardian
2. we monkeypatch until it works
3. we reset migrations (some additional dependency trickery might be necessary)

@Bartzi is there anything obvious i'm missing?

**Comment by janno42** [comment]
implemented in #666
